assets/alertmanager/pod-disruption-budget.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: policy/v1beta1
```

Comment:
Is this something we want, this was generated by bumping kube-prom I believe.

Reply:
I believe it is the fix for https://bugzilla.redhat.com/show_bug.cgi?id=1933847